apps/builder/app/builder/features/breakpoints/use-update-canvas-width.ts

```diff
@@ -36,7 +39,7 @@ export const useUpdateCanvasWidth = () => {
       : maxWidthBelowNextBreakpoint;
 
     const width = Math.max(
-      selectedBreakpoint.minWidth ?? 0,
+      "minWidth" in selectedBreakpoint ? selectedBreakpoint.minWidth ?? 0 : 0,
```

Comment:
Maybe no need to check for "in" operation. `selectedBreakpoint.minWidth ?? 0` look equal

Reply:
now yes, before I had a union type, a property was not available potentially so needed these checks